meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -95,8 +95,12 @@ do_provide_mender_defines() {
             ;;
     esac
 
-    # Strip leading and trailing whitespace, then newline divide, and remove dtbo's.
-    MENDER_DTB_NAME="$(echo "${KERNEL_DEVICETREE}" | sed -r 's/(^\s*)|(\s*$)//g; s/\s+/\n/g' | sed -ne '/\.dtbo$/b; p')"
+    if [ -n "${MENDER_MENDER_DTB_NAME_FORCE}" ]; then
```

Comment:
I think we can use the same variable name, MENDER_DBT_NAME.

In any case, let's call it MENDER_DBT_NAME_FORCE and avoid the MENDER_MENDER in the name.